prefect_aws/s3.py

```diff
@@ -337,9 +339,118 @@ def _get_s3_client(self) -> boto3.client:
             s3_client = self.aws_credentials.get_boto3_session().client(
```

Comment:
Should _get_s3_client be a public method so it can be used in the other s3 tasks?

Reply:
No, the tasks use `get_boto3_session` the credentials block that they are provided. We need this method because we have two different attributes for the two different credentials blocks.